pkg/mcs/tso/server/server.go

```diff
@@ -122,36 +120,11 @@ func CreateServerWrapper(cmd *cobra.Command, args []string) {
 	metricutil.Push(&cfg.Metric)
 
 	// TODO: Create the server
-	ctx, cancel := context.WithCancel(context.Background())
-	svr := &Server{}
-
-	sc := make(chan os.Signal, 1)
-	signal.Notify(sc,
-		syscall.SIGHUP,
-		syscall.SIGINT,
-		syscall.SIGTERM,
-		syscall.SIGQUIT)
-
-	var sig os.Signal
-	go func() {
-		sig = <-sc
-		cancel()
-	}()
-
-	if err := svr.Run(); err != nil {
-		log.Fatal("run server failed", errs.ZapError(err))
-	}
-
-	<-ctx.Done()
-	log.Info("Got signal to exit", zap.String("signal", sig.String()))
+	result := &bs.CreateServerResult{}
```

Comment:
Please ignore the hack. Working on a separate pr to init/start tso and will rewrite this function.